src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg, secondLeg, thirdLeg;
+    private String color;
+
+    public RightTriangle(double firstLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = firstLeg;
+        this.thirdLeg = firstLeg;
+        this.color = color;
+    }
+
+    public RightTriangle() {
+    }
+
+    @Override
+    public double findArea() {
+        return (firstLeg * firstLeg * Math.pow(3, 2)) / 4;
+    }
+
+    @Override
+    public String printAllInformation() {
+        return null;
```

Comment:
Returning `null` from a method is a bad practice. You should implement the `printAllInformation` method to return a string representation of the triangle, including its area and color.